.travis.yml

```diff
@@ -0,0 +1,11 @@
+matrix:
+  include:
+    - language: python
+      python: "3.6"
```

Comment:
Hi @Tomcli did you try Python 3.7 as well

Reply:
I can also add that to travis.